src/twisted/test/test_ssl.py

```diff
@@ -667,8 +671,10 @@ def test_method(self):
         # SSLv23_METHOD allows SSLv2, SSLv3, or TLSv1
         self.assertEqual(self.context._method, SSL.SSLv23_METHOD)
 
-        # And OP_NO_SSLv2 disables the SSLv2 support.
-        self.assertTrue(self.context._options & SSL.OP_NO_SSLv2)
+        # OpenSSL >= 1.1.0 no longer provides SSLv2
+        if not OPENSSL_1_1:
```

Comment:
> Earlier versions of OpenSSL could have SSLv2 compiled out too.

True, but [`SSL_OP_NO_SSLv2` is defined as zero as of 1.1.0](https://github.com/openssl/openssl/blob/OpenSSL_1_1_0/include/openssl/ssl.h#L339).  So it's fine to set it (`flags | 0` -> `flags`) but impossible to check (`flags & 0` -> `0`).